apps/dashboard/src/components/icons/version-control-dev.tsx

```diff
@@ -0,0 +1,30 @@
+import type { HTMLAttributes } from 'react';
+
+type VersionControlDevProps = HTMLAttributes<HTMLOrSVGElement>;
+export const VersionControlDev = (props: VersionControlDevProps) => {
```

Comment:
please run svgs through the: https://jakearchibald.github.io/svgomg/
and update the attributes from dashes to camel case for example `stroke-opacity` to `strokeOpacity`

Reply:
Very cool, didn't know about that